Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM python:3.9
```

Comment:
nit: blank line before the `FROM` line?

Also, and I do not know the right answer: should we use a SHA for the base image?

Reply:
Thanks for the callout here, I forgot to come back and add the SHA after getting things building.

I generally like pinning to a SHA for docker images so that any change to the base state is explicit. At a previous company we sometimes lost build stability because of seemingly minor changes in base images (perhaps base images are better citizens these days than they used to be?).